doc/service_def.json

```diff
@@ -0,0 +1,31 @@
+# Service Definition
```

Comment:
Should the file name be service_def.md instead of service_def.json?

Reply:
ah, good one. I have no idea why I named it .json, that's just wrong.